client/components/Pipeline.jsx

```diff
@@ -9,7 +9,9 @@ import CardContent from '@material-ui/core/CardContent';
 import * as Colors from '@material-ui/core/colors';
 import Typography from '@material-ui/core/Typography';
 
-import { showBuildLabels } from '../../app-config';
+import { goServerUrl, showBuildLabels, linkToPipelineInGo } from '../../app-config';
+
+const pipelineHistoryUrl = goServerUrl + '/go/tab/pipeline/history/';
```

Comment:
This does not work when you use env variable to setup the server. it will just use the default. The property needs to be loaded/communicated in runtime from the backend

Reply:
actuallty every property loaded by this approach is affected by this limitation